src/MyShop.Api/Program.cs

```diff
@@ -16,4 +17,5 @@
 
 var app = builder.Build();
 app.UseInfrastructure();
+app.MapGet("api", (IOptions<AppOptions> options) => Results.Ok(options.Value.Name));
```

Comment:
po co to?

Reply:
potrzebne do azure, dzieki temu bedzie dzialac, bo w appopsions przekazywany jest name, a on w appsettings jest przypisany do "Myshop Api "

Reply:
ale to nic nie daje do wywalenia